### PR TITLE
chore: updates README and adds CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by tdstein**
- **chore: updates README with additional instructions**
- **chore: adds CODE_OF_CONDUCT copied from https://github.com/posit-dev/great-tables/blob/main/CODE_OF_CONDUCT.md**
- **chore: updates README**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Contributor Covenant Code of Conduct
```
Comment (nealrichardson) [inline CODE_OF_CONDUCT.md:1]: For our future reference: the original source of this is https://github.com/r-lib/usethis/blob/main/inst/templates/CODE_OF_CONDUCT.md

File: README.md [hunk 2]
```diff
@@ -21,6 +21,18 @@ con = Client()
...
+
+If you encounter any issues or have any questions, please open an issue on the [issue tracker](https://github.com/posit-dev/posit-sdk-py/issues). We appreciate your feedback and will do our best to address any problems.
+
+## License
+
+This project is licensed under the [MIT License](LICENSE). Feel free to use, modify, and distribute the code as you see fit.
+
+## Code of Conduct
+
+Please note that this project follows the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). We expect all contributors to adhere to its guidelines and create a positive and inclusive community.
```
Comment (nealrichardson) [inline README.md:38]:
```suggestion
We expect all contributors to adhere to the project's [Code of Conduct](CODE_OF_CONDUCT.md) and create a positive and inclusive community.
```

File: README.md [hunk 3]
```diff
@@ -21,6 +21,18 @@ con = Client()
...
-<!-- Add notes about local development -->
+## Contributing
 
-<!-- Add code of conduct -->
\ No newline at end of file
+We welcome contributions to the Posit SDK for Python! If you would like to contribute, see the [CONTRIBUTING](CONTRIBUTING.md) guide for instructions.
+
+## Issues
+
+If you encounter any issues or have any questions, please open an issue on the [issue tracker](https://github.com/posit-dev/posit-sdk-py/issues). We appreciate your feedback and will do our best to address any problems.
```
Comment (nealrichardson) [inline README.md:30]:
```suggestion
If you encounter any issues or have any questions, please [open an issue](https://github.com/posit-dev/posit-sdk-py/issues). We appreciate your feedback.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -10,22 +10,20 @@ Before contributing to the `posit-sdk`, ensure that the following prerequisites
 
 - Python >=3.8
 
+## Instructions
```
Comment (nealrichardson) [inline CONTRIBUTING.md:13]: Have you confirmed that these instructions work?

**Review by nealrichardson — APPROVED** [review]

**Review by nealrichardson — APPROVED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (tdstein) [reply CONTRIBUTING.md:13]: I am running some tests to validate things work as expected on a forked repository.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -10,22 +10,20 @@ Before contributing to the `posit-sdk`, ensure that the following prerequisites
...
+
+1. Fork the repository and clone it to your local machine.
```
Comment (zackverham) [inline CONTRIBUTING.md:15]: does this imply that all PRs should come from forks moving forward? Less a question about the correctness of this stance, more want to make sure @dbkegley and I follow the expected pattern as we're working on things.

Comment (tdstein) [reply CONTRIBUTING.md:15]: Ah, good call out! For posit-dev members, direct pull requests are welcome!

**Pull request merged**